core-api/build.gradle

```diff
@@ -8,6 +8,8 @@ description = """ABSmartly Java SDK"""
 
 def httpClientVersion = "5.1.3"
 
+def lombokVersion = "1.18.26"
```

Comment:
Is it safe to use Lombok with old Android versions?

Reply:
Yeah, when I did it I forgot that was java 1.6 :(. Going to check.